src/main/java/com/facebook/openwifirrm/ucentral/IEUtils.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifirrm.ucentral;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+public abstract class IEUtils {
```

Comment:
This comment doesn't belong here specifically, but I wanted to leave this comment. Please add to `extractIEs` in `UCentralUtils` to actually extract these into `WifiScanEntry` via `InformationElements`.

Reply:
Are some of these fields present in wifi scans? If so, maybe we can add just those fields?

Reply:
@elludraon what do you think?

Reply:
Not as far as I can see - I checked a bunch and none of them were there. If you know where the schema is, that's another alternative (I tried looking in like every repo)